### PR TITLE
Assignment1 - Omar Facundo

### PR DESCRIPTION
**Pull request by omar5o9**
Fixed LoadQuestions  and changed the Calculations to float

### CODE REVIEW

**Comment by alyxadarr** [comment]
Reviewing :)

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs [hunk 1]
```diff
@@ -19,7 +19,7 @@ public static void Main(string[] args)
         Console.WriteLine("You got " + GetPercentCorrect(numberCorrect, questions.Length) + " correct");
     }
 
-    public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
+    public static string GetPercentCorrect(float numberCorrectAnswers, int numberOfQuestions)
```
Comment (alyxadarr) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:22]: I did not think of changing the method header for the test on the percent. What I did was cast it as a float in the below statement and did- return (float) (numberCorrectAnswers / numberOfQuestions * 100) + "%";
another good option but good job!

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs [hunk 2]
```diff
@@ -86,6 +86,7 @@ public static Question[] LoadQuestions(string filePath)
             question.Answers[1] = answer2;
             question.Answers[2] = answer3;
             question.CorrectAnswerIndex = correctAnswerIndex;
+            questions[i] = question;
```
Comment (alyxadarr) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:89]: I did the same thing here. Great job on the first Assignment.

**Review by alyxadarr — COMMENTED** [review]
Assignment Details
Pull Request title includes "Assignment 1" in the title ✔
Partner has also made a commit ❌ I don't think Omar had a partner for this assignment.
Pull Request targets Assignment 1 branch ✔
Issue 1: Application no longer crashes ✔
Issue 2: Unit test properly passes ✔
Extra Credit
Come up with a new feature for the application
Issue 3: Add a new feature request implemented ❌ Don't think a feature was added
Issue 3: Feature request unit tested ❌ Don't think feature was added

**Review by alyxadarr — APPROVED** [review]
Assignment Details
Pull Request title includes "Assignment 1" in the title ✔
Partner has also made a commit ❌ I don't think Omar had a partner for this assignment.
Pull Request targets Assignment 1 branch ✔
Issue 1: Application no longer crashes ✔
Issue 2: Unit test properly passes ✔
Extra Credit
Come up with a new feature for the application
Issue 3: Add a new feature request implemented ❌ Don't think a feature was added
Issue 3: Feature request unit tested ❌ Don't think feature was added

File: PrincessBrideTrivia/PrincessBrideTrivia/Program.cs (hunk 1 above)
Comment (robertogarcia11) [inline PrincessBrideTrivia/PrincessBrideTrivia/Program.cs:22]: aint no way

**Comment by MannyPoptart** [comment]
Looks good, you fixed the percentage function like I did.

Pull Request title includes "Assignment 1" in the title ✔
Partner has also made a commit: No partner
Pull Request targets Assignment 1 branch ✔
Issue 1: Application no longer crashes ✔
Issue 2: Unit test properly passes ✔
Extra Credit
Come up with a new feature for the application
Issue 3: Add a new feature request implemented ❌
Issue 3: Feature request unit tested ❌

**Pull request closed without merging**